create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java

```diff
@@ -53,7 +53,6 @@ public void setUp() {
     }
 
     @Test
-    @DisplayName("Request to Handler Returns Customer Created")
```

Comment:
No need to fix: Aren't display names easier to read?

Reply:
For me it's no difference really and I think it's more work to keep descriptions and test names in sync.